.github/workflows/testnet.yaml

```
@@ -0,0 +1,148 @@
+name: Testnet Tests
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+  pull_request:
+    branches:
+      - main
+      - dev
+  workflow_dispatch:
+    inputs:
+      skip_deployment:
+        description: "Skip deployment and use existing contracts"
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  testnet-tests:
+    name: Testnet Tests
+    runs-on: ubuntu-latest
+    env:
+      ENV: testnet
+      NODE_URL: https://aztec-alpha-testnet-fullnode.zkv.xyz
```

```suggestion
      NODE_URL: https://aztec-testnet-fullnode.zkv.xyz
```
should work without the alpha tag now. It still works with `alpha`, but we are moving away from that